include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -14,6 +14,19 @@
 
 namespace crocoddyl {
 
+/**
+ * @brief Sympletic RK4 integrator
```

Comment:
I don't understand in which sense the proposed integrator is symplectic.

Reply:
Hey @jcarpent -- thanks for pointing out this.
I wanted to say semi-implicit, not symplectic.
I have changed it (soon I will push it).

Let me know if you have further points :)

Reply:
I've seen the math written. Are you sure this is the semi-explicit version?
The derivatives are a bit harder than the pure explicit to write.

Maybe @andreadelprete could give some feedback on what he has implemented.